about.html

```diff
@@ -0,0 +1,182 @@
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="navbar main-nav navbar-expand-lg sticky-top">
+        <div class="container-md">
+            <a class="navbar-brand d-none d-sm-flex" href="#">
+                <img src="assets/images/Logo.png" alt="Afghan Team Logo" width="50px" height="60px">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse justify-content-end collapse-main" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                   <li class="nav-item">
+                        <a href="index.html" class="nav-link"> Home </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#logo" class="nav-link"> Programs </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#history" class="nav-link"> History </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link selected">About</a>
+                    </li>
+                    <li class="nav-item compaign">
+                        <a href="#" class="nav-link">CC Campaign</a>
+                    </li>
+                </ul>
+           </div>     
+        </div>
+    </nav>
+    </header>
+    <section class="intro-sec">
+        <div class="container-fluid">
+            <div class="container header-container d-flex flex-wrap align-content-center">
+                <div class="row justify-content-center m-3">
+                    <div class="col-md-12 intro-p abt">
+                        <h1>Afghanistan National Team 2023 Afghanistan Football Federation 1922 </h1>
+                    </div>
+                </div>
+
+                <div class="txt-border"></div>
+
+                <div class="row justify-content-center m-3">
+                    <div class="col-md-12 header-description abt-txt">
+                        <p>
+                            The Afghanistan national football team 
+                            is the national football team of Afghanistan and is controlled by the
+                            Afghanistan Football Federation. Founded in 1922, they played their first
+                            international game against Iran in Kabul, 1941. Afghanistan then joined
+                            FIFA in 1948 and the AFC in 1954, as one of the founding members.
+                            They play their home games at the Ghazi National Olympic Stadium in Kabul, 
+                            the capital of Afghanistan. In 2013, Afghanistan won the 2013 SAFF Championship
+                            and earned the "FIFA Fair Play Award". Afghanistan has never qualified for the AFC Asian Cup.
```

Comment:
- [ ] I would suggest reducing padding and sizes of images in your events section in about page to match the provided design

|Behance Design| Your's|
|---|---|
|Mobile screen:|---|
|<img width="453" alt="Screenshot 2023-06-28 at 10 39 50 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/b871697f-1fa5-4624-bda2-f38079aa9f76">|<img width="941" alt="Screenshot 2023-06-28 at 10 33 06 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/e528f066-7c01-438d-8965-1e659c3b5fb7">|
|Desktop Screen:|---|
|<img width="866" alt="Screenshot 2023-06-28 at 10 39 13 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/837cf5af-7ae3-42b2-8f56-acaae5b8d72c">|<img width="1426" alt="Screenshot 2023-06-28 at 10 39 24 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/dfa5443d-11e4-4978-be7e-24bf2f922b5e">|